qds_sdk/connection.py

```diff
@@ -78,23 +78,23 @@ def f_retry(self, *args, **kwargs):
             return f_retry  # true decorator
         return deco_retry
 
-    @retry((RetryWithDelay, requests.Timeout, ServerError, ApiThrottledRetry))
```

Comment:
@shekharsaurabh let's not change the names for existing exception classes. It has been there for a long time and there might be users who are specifically handling these exception classes. RetryWithDelay should not be changed